test/NaCl.Core.Benchmarks/Program.cs

```diff
@@ -8,14 +8,21 @@ class Program
     {
         static void Main(string[] args)
         {
+            var c = new ChaCha20Benchmark();
+            c.Size = 10_000;
+            c.Setup();
+            c.Encrypt();
```

Comment:
Do we need to call the benchmark like this? Maybe it was some test code that somehow got into here?